LICENSE

```diff
@@ -44,7 +44,7 @@
       of this License, Derivative Works shall not include works that remain
       separable from, or merely link (or bind by name) to the interfaces of,
       the Work and Derivative Works thereof.
- 
+
```

Comment:
@julianacook can you please revert the change above? It looks like there was a space that you removed (or maybe your editor did automatically). GitHub is parsing license files and they are rather particular about its content, so if possible I’d like you to revert that change above and only change the year. You can also close this PR and start another one, if that’s simpler for you? If you can’t help any more no worries, we can finish this up for you, just let us know :)